web/WEB-INF/web.xml

```diff
@@ -3,4 +3,68 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
          version="3.1">
-</web-app>
\ No newline at end of file
+
+    <!--<servlet>-->
```

Comment:
Don't use commented lines. Everything you want to save you can find in Version Control